native-script.rb

```diff
@@ -0,0 +1,42 @@
+# coding: utf-8
+
+puts "Installing Homebrew (a sudo password prompt may appear)."
+`ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+
+puts "Installing Homebrew-Cask (a sudo password prompt may appear)."
+`brew install caskroom/cask/brew-cask`
+
+puts "Installing Java (a sudo password prompt may appear)."
+`brew cask install java`
+
+puts "Creating NativeScript brew formula."
```

Comment:
Creating NativeScript formula...

Note that there are no mentions of brew formula in https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md#formula-cookbook It seems that this is not a legit term. If we do want to state that this is a Homebrew formula, then this should be "Creating NativeScript Homebrew formula..." or "Creating Homebrew formula for NativeScript..."